tests/e2e/env/e2e.sh

```diff
@@ -25,6 +25,14 @@ for arg in "$@"; do
 
 done
 
+# Remove the latest trailing slash from the URL
```

Comment:
🤏 A trailing slash = the latest `/`.

Reply:
https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2522/commits/d08282004ddcbc2cf70c68bc0e497c87615425fe